### DISCUSSION TITLE
How to duplicate a livewire component

### DISCUSSION THREAD

**Discussion by lucyacat** [laravel/framework#34740] [Q&A]
For the navigation dropdown, I want to duplicate and make another version for normal users(logged users will use different menus)
`@livewire('navigation-dropdown')`

I duplicated the blade(in the same folder), use it as `@livewire('navigation-dropdown-normal')`, but it doesn't work, the error message:
`app/Http/Livewire" directory does not exist`

Thanks a lot.

**Comment by lucyacat** [comment]
Sorry, I mean Jetstream

**Comment by RizONE05** [comment]
same issue (
Did you find out solution?

**Reply by lucyacat** [reply]
I created another livewire component instead, and put logic in that.
```
@livewire('navigation-bar', ['paramter' => true])

// navigation-bar.blade.php
@if (Auth::check())
...
````

Hope it will help you.

**Comment by qoelis** [comment]
Better create a Livewire component using php artisan...

php artisan make:livewire MyComponentName

2 files will be created : 

the first one in app\http\livewire\MyComponentName
the second in views directory

then you render your view in your component.
